docs/install-and-run.md

```diff
@@ -254,7 +254,7 @@ steps in your setup:
     #import <EarlGrey/EarlGrey.h>
    ```
 
-2. Download [`EarlGrey.swift`](https://github.com/google/EarlGrey/tree/master/gem/lib/earlgrey/files/EarlGrey.swift) and add it to your test bundle. The file contains
+2. Download [`EarlGrey.swift`](https://github.com/google/EarlGrey/tree/master/gem/lib/earlgrey/files/Swift-3.0/EarlGrey.swift) and add it to your test bundle. The file contains
```

Comment:
`swift/3.0` is more organized but it requires updating the gem. Since this is a quick fix, I would suggest linking to `https://github.com/google/EarlGrey/tree/master/gem/lib/earlgrey/files/` first then in a follow-up PR make the dir structure, gem and documentation changes.